lib/net/ssh/multi/session.rb

```diff
@@ -447,7 +450,7 @@ def io_select_wait(wait)
 
     def keepalive_interval
       servers = server_list.select { |s| s.options[:keepalive] }
-      servers.map { |s| s.options[:keepalive_interval] }.compact.min
+      servers.map { |s| s.options[:keepalive_interval] || DEFAULT_IO_SELECT_TIMEOUT }.compact.min
```

Comment:
I've added the method to Net::SSH in https://github.com/net-ssh/net-ssh/pull/343, and updated this pull request to use it.

We'll have to bump the minimum Net::SSH requirement here once a new version is released.